### ISSUE TITLE
feat: better (more pedagogical, aesthetic) screenshots

### ISSUE THREAD

**Issue by unusualevent** [cafkafk/eza#1099]
This project looks interesting.

The screenshot doesn't make it clear what each color means.

e.g., are things a different color when they're not checked into git?

It's a very messy set of colors.

**Comment by cafkafk** [comment]
> The screenshot doesn't make it clear what each color means.

I don't think we really document it anywhere either, but doing it by the color can be problematic because they're likely not gonna be the same for all users, so we'd have to document more just what color 0 means, what color 1 means, and so on. 

If we do have that documentation somewhere apologies, I do think exa had it and a overarching philosophy behind the various colors. If we're still compliant we could also just copy that.

> e.g., are things a different color when they're not checked into git?

If anyone want's to take better ones they should feel free to do so, the goal back then was to showcase eza features mostly, not colors, and was just like a very quick thing I did at the start of the project.

> It's a very messy set of colors.

These can be customized! And should just be a result of the colors you specify in your terminal in the first place, well, or wherever you specify colors.

**Comment by unusualevent** [comment]
ah. maybe it'd be better to showcase a directory where it's pretty straightforward to see what's ignored, what's a file edit, and what's new.

the first thing when looking at a screenshot I find is whether the results are intuitive. one of the screenshots does show git information (whatever -I is?) and the other one just jumps into lots of highlighting.
